### ISSUE TITLE
Midterm Paper Peer Review (Edmond Mui)

### ISSUE THREAD

**Issue by em656** [kathleenzhu/MedicalNoshow#7]
Overall, I enjoyed the graphics that were presented in the report and the initial data cleaning that you did. The report was well written (good balance of visuals and text) and seemed to do most of what a preliminary report would need.

Some of the things that I felt could be improved upon are:
1. Accuracy of your model. I find it somewhat concerning that you have only done a simple 80/20 split of your data into a training and test set that you only have an accuracy of roughly 70%. This is before you have conducted any measures to prevent overfitting. Either the logistic regression is not the right model for this project or you may have to look deeper into the features to make sure that your model can be as accurate as possible given the data. With that said, I am happy to see that you are heading in that direction from your Next Steps section.
2. This is more of a question and a challenge for the next steps of your project, but I was wondering how generalized your model can be. The data you selected is from Brazil and there are noticeable differences between Brazil and say, a country like the United States. Would you feel fairly confident that the model you derive from the Brazilian data could be changed slightly to work well for the American health care industry or do you believe that it would require starting from scratch and rebuilding a new model from American data?

I look forward to seeing the rest of your results.
